tests/integration/test_stores.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+
+@unittest.mock.patch("localstack.config.ALLOW_NONSTANDARD_REGIONS", True)
+def test_nonstandard_regions(create_boto_client, monkeypatch):
+    """
+    Ensure that non-standard AWS regions can be used vertically.
+    """
+    monkeypatch.setenv("MOTO_ALLOW_NONEXISTENT_REGION", "true")
```

Comment:
Ideally we want to patch the config option before LS starts. What's a proper way to do this?

Reply:
unfortunately there is no good way of doing as at all. especially not in a test. config.py seems good for the runtime